### PR TITLE
feat(connector-besu, connector-quorum): replace invokeContractV2

### LINKED ISSUE #679: feat(connector-besu): replace the invokeContract by the invokeContractV2
**Is your feature request related to a problem? Please describe.**
Replace the invokeContract function for the new invokeContractV2 and rename with the first one. All tests and examples should be work fine with this new function.

**Describe the solution you'd like**
The oldest invokeContract functionality must be replaced for the new one, the v2 must be deleted and only have v1. 
The changes must be made in the connectors but also in the tests and examples so that nothing fails

### PR DESCRIPTION
**Pull request by AzaharaC**
Resolve #679 

Depends on #702

### CODE REVIEW

File: packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/web-services/invoke-contract-endpoint.ts
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/web-services/invoke-contract-endpoint.ts:29]: @AzaharaC Thanks for fixing the occasional linter warning! It matters in the long run.

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/invoke-contract-v2.test.ts
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/invoke-contract-v2.test.ts:125]: @AzaharaC I'd also rename this file to drop the v2 because once we hit the v2 major release of Cactus (in the future) then this will collide name-wise with the tests that we'll add then.

**Review by petermetz — CHANGES_REQUESTED** [review]

File: examples/cactus-example-supply-chain-backend/src/main/typescript/infrastructure/supply-chain-app-dummy-infrastructure.ts
Comment (AzaharaC) [reply examples/cactus-example-supply-chain-backend/src/main/typescript/infrastructure/supply-chain-app-dummy-infrastructure.ts:145]: There is an issue to change this (#678). Jordi did a pr (#702) but it has not been accepted yet.

Comment (petermetz) [reply examples/cactus-example-supply-chain-backend/src/main/typescript/infrastructure/supply-chain-app-dummy-infrastructure.ts:145]: @AzaharaC Ohh, I see, then in the meantime you can either rebase onto Jordi's branch over there and do the updates here based on that OR you could just wait until that one gets merged and then rebase onto main. Either way, thanks for the reminder, I couldn't remember what was the status of that keychain change. I'll mark this as dependent on #702

File: packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts
Comment (AzaharaC) [reply packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts:205]: I have opened a new issue for this change. #743

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/invoke-contract-v2.test.ts
Comment (AzaharaC) [reply packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/invoke-contract-v2.test.ts:125]: done

File: packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/deploy-contract-from-json.test.ts
Comment (AzaharaC) [reply packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/deploy-contract-from-json.test.ts:412]: see pr #750

Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/deploy-contract-from-json.test.ts:412]: :+1: :+1: :+1:

**Review by petermetz — APPROVED** [review]

**Review by takeutak — APPROVED** [review]

**Comment by petermetz** [comment]
@AzaharaC @jordigiam Let me know if you need help with conflict resolution!

**Comment by petermetz** [comment]
@AzaharaC I did the rebase, now there's a failure that looks like a legit test failure that's worth looking into => https://github.com/hyperledger/cactus/pull/731/checks?check_run_id=2383092323#step:5:14645

**Comment by AzaharaC** [comment]
> @AzaharaC I did the rebase, now there's a failure that looks like a legit test failure that's worth looking into => https://github.com/hyperledger/cactus/pull/731/checks?check_run_id=2383092323#step:5:14645

I'm going to check this error

**Pull request merged**
